### PR TITLE
Fix indentation

### PR DESCRIPTION
**Pull request by OrKoN**
This PR fixes the indentation issues and reworks tests to test `stringifyStep` and `stringify` separately using @puppeteer/replay. This removes the need for the LineWriterImpl. Also, removes Puppeteer as a dependency as it should not be needed.

### CODE REVIEW

**Comment by OrKoN** [comment]
cc @nagash77

File: package.json [hunk 1]
```diff
@@ -31,14 +31,13 @@
...
   "dependencies": {
-    "@puppeteer/replay": "^2.6.1",
+    "@puppeteer/replay": "^2.10.2",
     "chalk": "^5.0.1",
     "execa": "^6.1.0",
     "globby": "^13.1.1",
     "inquirer": "^8.2.1",
     "is-git-clean": "^1.1.0",
-    "meow": "^10.1.2",
-    "puppeteer": "^19.2.2"
```
Comment (emilyrohrbough) [inline package.json:41]: Why was puppeteer removed to fix indentation?

**Comment by emilyrohrbough** [comment]
@OrKoN Thank you for the contribution! Can you link the indentation issues you are referencing? Also can you provide before & after screenshots of this fix?

File: package.json (hunk 1 above)
Comment (OrKoN) [reply package.json:41]: Apologies, I removed it because it is not used and it's not related to the indentation fix.

**Comment by OrKoN** [comment]
Here is a screenshot of the current version of the Cypress extension: `describe/it` are not indented. For the 'after' version, you can check the test I have added "correctly stringifies an entire recording"

Screenshot 2023-03-29 at 08 26 23

**Review by BadKarma187 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (emilyrohrbough) [reply package.json:41]: @OrKoN It looks like puppeteer is a[ peer dependency](https://github.com/puppeteer/replay/blob/main/package.json#L89) of `@puppeteer/replay` so we need to keep this.

Comment (OrKoN) [reply package.json:41]: It’s an optional peer dependency and it should only be needed for create Puppeteer-based runner extensions. As far as I see this module offers only a stringify extension. Is there a runner extension that I have overlooked?

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**